dbms/src/Debug/dbgFuncCoprocessor.cpp

```diff
@@ -339,56 +340,99 @@ void dbgFuncTiDBQueryFromNaturalDag(Context & context, const ASTs & args, DBGInv
     auto dag = NaturalDag(json_dag_path, &Poco::Logger::get("MockDAG"));
     dag.init();
     dag.build(context);
-    bool unequal_flag = false;
-    String unequal_msg;
+    std::vector<std::pair<int32_t, String>> failed_req_msg_vec;
```

Comment:
```suggestion
    std::vector<std::pair<Int32, String>> failed_req_msg_vec;
```
Maybe better?

Reply:
They seem the same to me:) And both usages can be found in our projects.